packages/button-react/src/Button.tsx

```diff
@@ -1,27 +1,27 @@
-import React, { MouseEventHandler, ReactNode } from "react";
+import React, { HTMLAttributes } from "react";
 import classNames from "classnames";
 
-interface Props {
-    children: ReactNode;
-    className?: string;
-    onClick?: MouseEventHandler<HTMLButtonElement>;
+interface Props extends Exclude<HTMLAttributes<HTMLButtonElement>, "disabled"> {
```

Comment:
Arver alle egenskaper fra `<button>` bortsett fra disabled. Er vel en grei løsning på kommentarene deres @Saegrov @lfbergee ?

Reply:
Ja. Må gjerne lage en issue for disabled også for det ser vi behov for i Flyt.